test/WebJobs.Script.Tests/Workers/Rpc/RpcFunctionInvocationDispatcherTests.cs

```diff
@@ -126,6 +126,31 @@ public async Task WorkerIndexing_Setting_ChannelInitializationState_Succeeds()
             Assert.Equal(expectedProcessCount, initializedChannelsCount);
         }
 
+        [Fact]
+        public async Task WorkerRuntime_Setting_ChannelInitializationState_Succeeds()
```

Comment:
please add variation of this test - via `[Theory]` - one with env variable set and one without

Reply:
Done